src/main/java/lotto/domain/result/Result.java

```diff
@@ -0,0 +1,32 @@
+package lotto.domain.result;
+
+import lotto.domain.lottery.Lottery;
+import lotto.domain.lottery.WinningLottery;
+import lotto.domain.rank.Rank;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Result {
+    private final Map<Rank, Long> result = new HashMap<>();
```

Comment:
```suggestion
    private final Map<Rank, Long> result = new EnumMap<>(Rank.class);
```
닉이 이슈에 등록한 부분인데 EnumMap 사용해주면 좋을거 같아요!